core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -81,11 +158,7 @@ function ampContent() {
         // errors in video, because video will be stripped out.
         // @TODO: remove this, when Amperize support video transform
         $('video').children('source').remove();
-
-        // Vimeo iframe e. g. come with prohibited attributes
-        // @TODO: remove this, when Amperize supports HTML sanitizing
-        $('amp-iframe').removeAttr('webkitallowfullscreen');
-        $('amp-iframe').removeAttr('mozallowfullscreen');
+        $('video').children('track').remove();
```

Comment:
No, because of `amp-*: ['*']` which would allow further amp-HTML tags and allow all remaining attributes. But style is one, that always has to be stripped out.